cmd/flags/fail_readiness.go

```diff
@@ -0,0 +1,36 @@
+//Copyright 2019 Expedia, Inc.
+//
+//Licensed under the Apache License, Version 2.0 (the "License");
+//you may not use this file except in compliance with the License.
+//You may obtain a copy of the License at
+//
+//http://www.apache.org/licenses/LICENSE-2.0
+//
+//Unless required by applicable law or agreed to in writing, software
+//distributed under the License is distributed on an "AS IS" BASIS,
+//WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//See the License for the specific language governing permissions and
+//limitations under the License.
+
+package flags
+
+import (
+	"flag"
+	"fmt"
+)
+
+type FailReadiness struct {
+	Enabled      bool `json:"fail-readiness-enabled"`
```

Comment:
I think this "enabled" makes the usage more complicated.
If users don't want to fail the readiness checks they can simply not enable any of the individual checks.